dist/travisci/.travis/RoboFile.php

```diff
@@ -235,7 +235,7 @@ protected function runUnitTests()
             ->copy('.travis/config/phpunit.xml', 'web/core/phpunit.xml', $force);
         $tasks[] = $this->taskExecStack()
             ->dir('web')
-            ->exec('../vendor/bin/phpunit -c core --debug --coverage-clover ../build/logs/clover.xml --verbose modules/custom');
```

Comment:
getting segmentation fault and nesting level limits reached with xdebug. not requiring it to run the test.